lib/main_common.pm

```diff
@@ -1409,6 +1409,9 @@ sub load_extra_tests_y2uitest_gui {
 sub load_extra_tests_y2uitest_cmd {
     loadtest 'yast2_cmd/yast_lan';
     loadtest 'yast2_cmd/yast_timezone';
+    if (is_sle('15+')) {
+        loadtest 'yast2_cmd/yast_rdp';
+    }
```

Comment:
just hint, I guess people like more one liners ^^
https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/7765/files#diff-d3e0a6c7df772cff7741832a6250d5afL1405

Reply:
yes, it looks better.